apps/explorer/src/index.tsx

```diff
@@ -30,7 +32,22 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
             <QueryClientProvider client={queryClient}>
                 <CookieManagerProvider>
                     <RouterProvider router={router} />
-                    <CookieDisclaimer />
+                    <Disclaimer
+                        onClose={() => {
+                            handleConsentAccepted();
+                        }}
+                    >
+                        <div className="text-neutral-10 dark:text-neutral-92 text-body-md">
```

Comment:
```suggestion
                        <div className="text-iota-neutral-10 dark:text-iota-neutral-92 text-body-md">
```
I think the text colors might be off, we should use the UI kit color palette.